public/components/event_analytics/explorer/events_views/data_grid.tsx

```diff
@@ -114,8 +123,12 @@ export function DataGrid(props: DataGridProps) {
   const dataGridColumns = () => {
     const columns: EuiDataGridColumn[] = [];
     selectedColumns.map(({ name }) => {
-      if (name === 'timestamp') {
-        columns.push(DEFAULT_TIMESTAMP_COLUMN);
+      if (name === defaultTimestamp) {
+        columns.push({
+          ...DEFAULT_TIMESTAMP_COLUMN,
+          display: `Time (${defaultTimestamp})`,
```

Comment:
we need i18n for all these user facing text

Reply:
implemented